gtk4/src/media_stream.rs

```diff
@@ -0,0 +1,35 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::MediaStream;
+use glib::translate::*;
+use glib::IsA;
+
+pub trait MediaStreamExtManual: 'static {
+    #[doc(alias = "gtk_media_stream_error")]
+    fn error(&self, domain: glib::Quark, code: i32, message: &str);
+
+    #[doc(alias = "gtk_media_stream_gerror")]
+    fn gerror(&self, error: &glib::Error);
+}
+
+impl<O: IsA<MediaStream>> MediaStreamExtManual for O {
+    fn error(&self, domain: glib::Quark, code: i32, message: &str) {
+        unsafe {
+            ffi::gtk_media_stream_error(
+                self.as_ref().to_glib_none().0,
+                domain.to_glib(),
+                code,
+                message.to_glib_none().0,
+            )
+        }
+    }
+
+    fn gerror(&self, error: &glib::Error) {
+        unsafe {
+            ffi::gtk_media_stream_gerror(
+                self.as_ref().to_glib_none().0,
+                mut_override(error.to_glib_full()),
```

Comment:
And do the usual workaround to steal the `GError` and pass it to this function :)